woocommerce/woocommerce-android
# Issue/2844 order detail refactor shipping labels card

This PR takes care of the 9/13 step of the Order Detail refactoring #2844 .

#### Changes
- Adds the Shipping labels card (`OrderDetailShippingLabelsView`) to the Order detail screen. This card will be visible only in debug build and when the WooCommerce Services plugin is available for the store.

#### Notes
- I have split this massive change into multiple small PRs in order to make reviews easier. 
- This PR is to be merged into a feature branch.
- **This PR is in draft till this [Step 8 PR](https://github.com/woocommerce/woocommerce-android/pull/2873) can be merged.**

#### Testing
- Click on an order from the Orders tab that has some shipping labels associated with it. (To create shipping labels, please follow the instructions provided in the Shipping Labels Master Thread - `p91TBi-2om`)
- Notice the shipping labels section displayed.


#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/93732166-c4745280-fbed-11ea-9b4b-796cfb022bec.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/93732169-c63e1600-fbed-11ea-8ce5-85880a385822.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/93732187-d655f580-fbed-11ea-872a-3a940827c9e6.png" width="300"/>. <img src="https://user-images.githubusercontent.com/22608780/93732190-d8b84f80-fbed-11ea-9cfe-19587bc6ec8e.png" width="300"/>

Update release notes:
